frontend/src/app/AppLayout/AppLayout.tsx

```diff
@@ -18,6 +18,7 @@ import {
 } from '@patternfly/react-core';
 import { IAppRoute, IAppRouteGroup, routes } from '@app/routes';
 import { BarsIcon } from '@patternfly/react-icons';
+import logo from '../bgimages/bot_avatar.svg';
```

Comment:
nit: Not need to change this but just afaik, it would be great to store images in an `image` folder and reference it with an absolute path.